pyproject.toml

```diff
@@ -48,6 +48,8 @@ dependencies = [
     "ruff",
     # for packaging.version; not sure what the lower bound is.
     "packaging",
+    # for jupyter notebook export
+    "nbformat>=5.7.0",
```

Comment:
Should we add this to recommended instead?

Reply:
yea recommended makes sense